docs-source/content/release-notes.md

```diff
@@ -8,7 +8,7 @@ draft: false
 
 | Date | Version | Introduces backward incompatibilities | Change |
 | --- | --- | --- | --- |
-| July 1, 2020 | v3.0.0 | yes | Adds Model in Image feature and support for Istio. Removal of support for Helm 2.x. Operator performance improvements to manage many domains in the same Kubernetes cluster. |
+| July 17, 2020 | v3.0.0 | yes | Adds Model in Image feature and support for Istio. Removal of support for Helm 2.x. Operator performance improvements to manage many domains in the same Kubernetes cluster. |
```

Comment:
It is missing these two important features/certifications

OLCNE 1.1 certification, 

For Domain Home on PV added the ability to apply topology changes to the domain which the Operator can now uptake without requiring any downtime.

Reply:
We added this to the 2.6.0 release notes, "Certified support of Oracle Linux Cloud Native Environment (OLCNE) 1.1 with Kubernetes 1.17.0."

Reply:
Monica, I'll add a statement about applying topology changes. Is it okay that the OLCNE certification is just on the 2.6.0 release? It's okay with me if we copy it over.

Reply:
yes